app/src/main/java/com/humberto/tasky/agenda/presentation/agenda_details/AgendaDetailsScreen.kt

```diff
@@ -338,6 +343,11 @@ private fun AgendaDetailsScreen(
                 color = MaterialTheme.colorScheme.tertiary
             )
             if(agendaItem is AgendaItemDetails.Event) {
+                AddEventAttendeeDialog(
+                    onDismiss = { onAction(AgendaDetailsAction.OnDismissAttendeeDialog) },
+                    onAdd = { onAction(AgendaDetailsAction.OnAddAttendeeClick) },
+                    agendaItem = agendaItem
+                )
                 Column(
```

Comment:
Would a boolean state not make sense here to manage the visibility of this dialog? It appears it will be rendered when the user lands on this screen (or is that intended behaviour for testing? 🤔 )